eng/Publish-MavenPackages.ps1

```diff
@@ -0,0 +1,294 @@
+param(
```

Comment:
For consistency it might be better to move this into a `eng\scripts` directory.

Reply:
Should I replicate this pattern for all the other Repos?

Reply:
I mean  do we want to eventually move all the freely hanging script files into a script directory for all the repos?